src/main/java/org/springframework/shell2/standard/ShellOption.java

```diff
@@ -27,19 +27,20 @@
  *
  * @author Eric Bottard
  * @author Florent Biville
+ * @author Camilo Gonzalez
  */
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
-@Target(ElementType.PARAMETER)
+@Target({ ElementType.PARAMETER, ElementType.FIELD })

```

Comment:
added in preparation to allow for @ShellOption inside "argument" classes similar to JCommander, e.g.:

```java
@ShellMethod(...)
public void doSomething(ArgumentHolder args) {...}

private static class ArgumentHolder {
   @ShellOption(...)
   private String argument1;
}
```